client/web/src/site-admin/SiteAdminPackagesPage.tsx

```diff
@@ -38,95 +45,96 @@ import {
 
-const ExternalServicePackageMap: Partial<
-    Record<
-        ExternalServiceKind,
-        {
-            label: string
-            value: PackageRepoReferenceKind
-        }
-    >
-> = {
-    [ExternalServiceKind.NPMPACKAGES]: {
-        label: 'NPM',
-        value: PackageRepoReferenceKind.NPMPACKAGES,
-    },
-    [ExternalServiceKind.GOMODULES]: {
-        label: 'Go',
-        value: PackageRepoReferenceKind.GOMODULES,
-    },
-    [ExternalServiceKind.JVMPACKAGES]: {
-        label: 'JVM',
-        value: PackageRepoReferenceKind.JVMPACKAGES,
-    },
-    [ExternalServiceKind.RUBYPACKAGES]: {
-        label: 'Ruby',
-        value: PackageRepoReferenceKind.RUBYPACKAGES,
-    },
-    [ExternalServiceKind.PYTHONPACKAGES]: {
-        label: 'Python',
-        value: PackageRepoReferenceKind.PYTHONPACKAGES,
-    },
-    [ExternalServiceKind.RUSTPACKAGES]: {
-        label: 'Rust',
-        value: PackageRepoReferenceKind.RUSTPACKAGES,
-    },
-}
-
 interface PackageNodeProps {
     node: SiteAdminPackageFields
+    setFilterPackage: (node: SiteAdminPackageFields) => void
 }
 
-const PackageNode: React.FunctionComponent<React.PropsWithChildren<PackageNodeProps>> = ({ node }) => {
+const PackageNode: React.FunctionComponent<React.PropsWithChildren<PackageNodeProps>> = ({
+    node,
+    setFilterPackage,
+}) => {
     const PackageIconComponent = externalRepoIcon({ serviceType: node.kind })
 
     const packageRepository = node.repository
 
     return (
         <li className="list-group-item px-0 py-2">
-            <div className={styles.node}>
-                <div>
-                    <Icon as={PackageIconComponent} aria-label="Package host logo" className="mr-2" />
-                    {packageRepository ? (
-                        <>
-                            <RepoLink repoName={node.name} to={packageRepository.url} />
-                            <RepoMirrorInfo mirrorInfo={packageRepository.mirrorInfo} />
-                        </>
-                    ) : (
-                        <>{node.name}</>
-                    )}
+            <div>
+                <div className={styles.node}>
+                    <div>
+                        <Icon as={PackageIconComponent} aria-label="Package host logo" className="mr-2" />
+                        {packageRepository ? (
+                            <>
+                                <RepoLink repoName={node.name} to={packageRepository.url} />
+                                <RepoMirrorInfo mirrorInfo={packageRepository.mirrorInfo} />
+                            </>
+                        ) : (
+                            <>
+                                <span>{node.name}</span>
+                                <Text className="mb-0 text-muted">
+                                    <small>This package has not yet been synced.</small>
+                                </Text>
+                            </>
+                        )}
```

Comment:
```suggestion
                        ) : (node.blocked ? (
				    <>
	                                <span>{node.name}</span>
	                                <Text className="mb-0 text-muted">
	                                    <small>This package is blocked by a filter.</small>
	                                </Text>
	                            </>
                        	) : (
	                            <>
	                                <span>{node.name}</span>
	                                <Text className="mb-0 text-muted">
	                                    <small>This package has not yet been synced.</small>
	                                </Text>
	                            </>
	                        )}
	                    )
```

Spacing is completely fecked because of the text box I tried :sweat_smile: